scss/datetime/_layout.scss

```diff
@@ -118,4 +139,95 @@
 
     }
 
+    // Infinite timepicker
+    // Header
+    .k-time-header { // Same as the calendar header
```

Comment:
If the header is the same by design, perhaps we can use the `k-calendar-header` class? There are also duplicated styles in the _theme file, with k-time-now acting as k-today.